### ISSUE TITLE
ENS: ethart.dfohub.eth directs to 404

### ISSUE THREAD

**Issue by ghost** [b-u-i-d-l/ethArt#2]
![ethartdfohub](https://user-images.githubusercontent.com/35188616/85197967-f77f1100-b2a1-11ea-957f-283d68b135a6.png)

On the ethart DFOhub the ens link goes to a 404 not found.

**Comment by alet89** [comment]
Yes, I see, because the ENS is set to https://www.b-u-i-d-l.github.io/ethart a web2 link and the frontend function of DFOhub is designed to work for IPFS, SWARM or other distributed link... 

Basically, if you use Github pages (a function in Github to build a web2 link from the Github repo) and you add a custom domain (in this case www.ethart.org) the original Github pages link (b-u-i-d-l.github.io/ethart) creates some redirect problems. 

BTW right now, Ethart is set on a web2 link, after 1 month of fixing and improving some frontend functions we'll deploy Ethart on IPFS, so you'll be able to reach it by the ENS.
